api/src/main/java/com/epam/pipeline/manager/datastorage/DataStorageManager.java

```diff
@@ -343,6 +353,40 @@ public AbstractDataStorage updatePolicy(DataStorageVO dataStorageVO) {
         return createdStorage;
     }
 
+    @Transactional(propagation = Propagation.REQUIRED)
+    public AbstractDataStorage createDefaultStorageForUser(final String userName) {
```

Comment:
Can we return `Optional` from this method, as it returns null in some cases?

Reply:
Done